### DISCUSSION TITLE
Errors in Laravel after install

### DISCUSSION THREAD

**Discussion by mihainn** [spatie/laravel-ray#134] [Q&A] [answered]
Hi, after install laravel-rey and move code o other server, i receive some errors in Laravel. [Stack link](https://stackoverflow.com/questions/65950888/laravel-8-project-is-not-opening-in-the-server-getting-error-in-testdatabases-p/66032033#66032033)

I not figured out if problem is from server (Homestead), PHP version(7.4.7), Laravel(8.26.1) or Ray package :\
Any ideas?

**Comment by freekmurze** [comment, marked as answer]
A syntax error in most cases means that you are not running the PHP version you think are running. Our tests prove that our package does not contain any syntax errors.
